src/daft-io/src/s3_like.rs

```diff
@@ -118,6 +120,23 @@ enum Error {
     UploadsCannotBeAnonymous {},
 }
 
+const THROTTLING_ERRORS: &[&str] = &[
+    "Throttling",
+    "ThrottlingException",
+    "ThrottledException",
+    "RequestThrottledException",
+    "TooManyRequestsException",
+    "ProvisionedThroughputExceededException",
+    "TransactionInProgressException",
+    "RequestLimitExceeded",
+    "BandwidthLimitExceeded",
+    "LimitExceededException",
+    "RequestThrottled",
+    "SlowDown",
+    "PriorRequestNotComplete",
+    "EC2ThrottledException",
+];
```

Comment:
Where did this come from? Should we document this list somehow

Reply:
done